taxi/tests.py

```diff
@@ -1,3 +1,76 @@
 from django.test import TestCase
+from taxi.models import Manufacturer, Driver, Car
 
-# Create your tests here.
+
+class ManufacturerModelTests(TestCase):
+    # Тест на створення виробника
+    def test_manufacturer_creation(self):
+        manufacturer = Manufacturer.objects.create(
+            name="Toyota", country="Japan"
+        )
+        self.assertEqual(str(manufacturer), "Toyota Japan")
+        self.assertEqual(manufacturer.name, "Toyota")
+        self.assertEqual(manufacturer.country, "Japan")
+
+    # Тест на унікальність назви виробника
+    def test_manufacturer_unique_name(self):
+        Manufacturer.objects.create(name="Toyota", country="Japan")
+        with self.assertRaises(Exception):
+            Manufacturer.objects.create(
+                name="Toyota", country="USA"
+            )
+
+    # Тест на створення водія з іменем та прізвищем
+    def test_driver_creation_with_names(self):
+        driver = Driver.objects.create_user(
+            username="johndoe", password="testpass", license_number="AB12345",
+            first_name="John", last_name="Doe"
+        )
+        self.assertEqual(str(driver), "johndoe (John Doe)")
+
+    # Тест на правильність URL для водія
+    def test_driver_get_absolute_url(self):
+        driver = Driver.objects.create_user(
+            username="johndoe", password="testpass", license_number="AB12345"
+        )
+        self.assertEqual(driver.get_absolute_url(), f"/drivers/{driver.pk}/")
```

Comment:
The expected URL in `test_driver_get_absolute_url` does not match the URL pattern defined in the `get_absolute_url` method of the `Driver` model. The test expects `/drivers/{driver.pk}/`, but the actual pattern is `reverse('taxi:driver-detail', kwargs={'pk': self.pk})`. Ensure the URL pattern in the test matches the actual implementation.